AElf.OS.Network.Grpc/GrpcPeerPool.cs

```diff
@@ -134,7 +134,7 @@ public GrpcPeer FindPeer(string peerEndpoint, byte[] publicKey = null)
             IEnumerable<GrpcPeer> toFind = _authenticatedPeers;
 
             if (!string.IsNullOrWhiteSpace(peerEndpoint))
-                toFind = toFind.Where(p => p.PeerAddress == peerEndpoint);
+                toFind = toFind.Where(p => p.PeerAddress == peerEndpoint || peerEndpoint.EndsWith(p.RemoteListenPort));
```

Comment:
do not use port to match peer.

just public key

Reply:
will be solved by #940